### ISSUE TITLE
Documentation for Positions parameter of the Cycle Position command needs improvement

### ISSUE THREAD

**Issue by duckpodger** [digitalcreations/MaxTo#750]
The documentation says "Accepts a list of relative rectangles, separated by pipe characters". There is no detail as to how to specify a "relative rectangle". 

**Is your feature request related to a problem? Please describe.**
The "cycle position" command does what I want on my landscape monitor, e.g. allowing me to cycle between the left third, left half, left two thirds when using the "left" preset. However, the "top" preset cycles between the top half of the screen and the middle third of the top half of the screen. 

**Describe the solution you'd like**
I would like to be able to setup a "cycle position" command that would cycle between the top third, top half, and top two thirds. This would give me the equivalent behaviour to the "left" preset for my portrait screen. I would call this "portrait top" if I could give it a name.

I think that the positions parameter would allow me to achieve this, but without more details on how to configure  it I cannot work out how

**Describe alternatives you've considered**
I have tried comma separated pairs of integers, percentages and decimal forms e.g.:

1,2|2,1
50%,50%|25%,75%
0.5,0.5|0.25,0.75

None have worked. 

**Additional context**
To be honest with you I have been using [Rectangle](https://rectangleapp.com/) on Mac recently, but after changing jobs I want similar keyboard shortcuts to those it enables. It re-orientates what the first third means based on the screen orientation. I find that when I am debugging in chrome having it running on the bottom two thirds allows me to have e.g. slack in the first third, and chrome in the last two thirds (the page uses half of that, and the debug tools the other half, i.e. I would take one third of the screen each). When I am not debugging I could reduce chrome to use the middle third and have another window in the bottom third.

Another alternative would be for the expand/contract edge command to take an optional parameter to indicate how much to expand by, because the hard coded amount takes too many keystrokes.

Switching between region presets and then moving the windows is also a bit of a pain.

**Comment by duckpodger** [comment]
p.s. Your product is very nearly doing what I want, but without being able to cycle positions as I describe (top third, top half, top two thirds), I will probably chose to use power toys fancy zones instead once the free trial expires.

**Comment by vegardlarsen** [comment]
@duckpodger I agree that this isn't documented well enough; exposing this argument at all was an afterthought.

Here are the definitions for the defaults it allows (meaning if you use `/Preset` instead of `/Positions`):

```cs
            { GridPosition.TopLeft, "0,0,50%,50%|0,0,33.333333%,50%|0,0,66.666666%,50%" },
            { GridPosition.Top, "0,0,100%,50%|33.333333%,0,33.333333%,50%" },
            { GridPosition.TopRight, "50%,0,50%,50%|66.666666%,0,33.333333%,50%|33.333333%,0,66.666666%,50%" },
            { GridPosition.Left, "0,0,50%,100%|0,0,33.333333%,100%|0,0,66.666666%,100%" },
            { GridPosition.Center, "0,0,100%,100%|33.333333%,0,33.333333%,100%|25%,0,50%,100%" },
            { GridPosition.Right, "50%,0,50%,100%|66.666666%,0,33.333333%,100%|33.333333%,0,66.666666%,100%" },
            { GridPosition.BottomLeft, "0,50%,50%,50%|0,50%,33.333333%,50%|0,50%,66.666666%,50%" },
            { GridPosition.Bottom, "0,50%,100%,50%|33.333333%,50%,33.333333%,50%" },
            { GridPosition.BottomRight, "50%,50%,50%,50%|66.666666%,50%,33.333333%,50%|33.333333%,50%,66.666666%,50%" }
```

As you can see, rectangles are defined as `x,y,w,h`, and the rectangles are `|`-separated. Coordinates can be either absolute (no unit or `px`) or relative to the monitor (`%`).

**Comment by duckpodger** [comment]
Awesome. Is the % always optional? or only omitted for a 0 value? No need
to answer right away now. I can have a play with it myself.

Out of interest are you on the windows insider program? Do you know how
irrelevant your code will be with the improved window management
capabilities in windows 11?

**Comment by duckpodger** [comment]
Sorry I forgot the "thank you" after the "awesome"!

**Comment by vegardlarsen** [comment]
Sorry for being late responding to this. No unit = `px`. `%` is measured in percent relative to the monitor.
